rocketpy/mathutils/function.py

```diff
@@ -22,6 +22,8 @@
     RBFInterpolator,
 )
 
+from ..plots.plot_helpers import show_or_save_plot
```

Comment:
it warns me a bit that you are creating the first rocketpy dependency on the Function class.
Ideally, the Function class should use no function or method from the rocketpy package. Do you agree?

Do we really need this function here? What would be the negative consequences if we exclude the Function class from this PR?

Reply:
Given that this import is essentially a "standalone tool", I don't see that being a huge problem compared to, let's say, we imported a `staticmethod` from the `Flight`. One could simply move the whole show or save function to the `Function` module and import from there to the rest of rocketpy.

The import is used to get the show or save functionality shortcut to the `Function` class, e.g.:

```py
my_motor.thrust() # plots, as usual
my_motor.thrust(filename="thrust_curve.png") # saves the plot
```

Reply:
Ok, agreed. 

#665 related.